### ISSUE TITLE
Unable to custom the MAUI Android Talkback configuration in the keyboard

### ISSUE THREAD

**Issue by rubenmorameneses** [dotnet/maui#26002]
Unable to custom the Talkback speeches configuration when a keyboard key is pressed. For example, if I press number one key in a number type entry, I heard something like "uno viñeta" (I'm spanish developer), I would like to perform like this "tecla uno presionada" or another custom string.
